tests/test_cli/test_escape.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+
+def test_quote_escape(monkeypatch, delegator):
+    """Check that cli works with multiline inputs"""
+    monkeypatch.setenv('MULTILINE_VALUE', json.dumps({
+        'key': 'value',
+        'key2': 'multi\nline\nvalue',
+    }, indent=4))
+
+    variables = delegator('dump-env -p MULTILINE_')
+    assert variables == '''VALUE="{
```

Comment:
these fixtures would actually be expressed in a more readable manner via a snapshot test facility, for example https://github.com/syrupy-project/syrupy - let me know if you want me to refactor these to use snapshots instead.

Reply:
Please, use `syrupy`!

Reply:
done in c2c8e943b0a1fe914d12767d8570d62566293779